### ISSUE TITLE
Change logger level

### ISSUE THREAD

**Issue by AlfredoRamos** [Nakilon/reddit_bot#10]
Hi,

First of all, thanks for creating this gem and keep it updated.

I recently migrated from another gem, and everything is working nicely so far however I would like to change the level of the `logger` but it seems that currently it can't be changed.

https://github.com/Nakilon/reddit_bot/blob/f524337f700618b1525de00016d4774748eed8e0/lib/reddit_bot.rb#L13-L17

In my application I've set it to `Logger::ERROR`, and from time to time my logs get populated with `info` messages that I would like to get rid of, specially this one:

https://github.com/Nakilon/reddit_bot/blob/f524337f700618b1525de00016d4774748eed8e0/lib/reddit_bot.rb#L214

Would it be possible to allow to set the logger level?

**Comment by Nakilon** [comment]
Hi!

Did not expect anyone would use it, heh. It was pushed to Github mostly for some demanding OPs in /r/RequestABot.

How do you change the level in your app?

```
$ cat Gemfile
source "https://rubygems.org"
gem "reddit_bot"
$ bundle

$ bundle exec irb
irb(main):001:0> require "yaml"
=> true
irb(main):002:0> require "reddit_bot"
=> true
irb(main):003:0> bot = RedditBot::Bot.new YAML.load_file "secrets.yaml"
=> #<RedditBot::Bot:0x007fc155101030 ...>
irb(main):004:0> bot.new_posts("r/largeimages").take(10).size
I, [2018-12-09T21:01:16.100321 #15375]  INFO -- : new token is: ...
D, [2018-12-09T21:01:17.840742 #15375] DEBUG -- : x-ratelimit-remaining=...
=> 10
irb(main):005:0> bot.new_posts("r/largeimages").take(10).size
D, [2018-12-09T21:01:22.074229 #15375] DEBUG -- : x-ratelimit-remaining=...
=> 10
irb(main):006:0> RedditBot.logger.level = Logger::ERROR
=> 3
irb(main):007:0> bot.new_posts("r/largeimages").take(10).size
=> 10
```

It seems to work for me here.

**Comment by AlfredoRamos** [comment]
Oh, I see, that's great!

> Did not expect anyone would use it, heh. It was pushed to Github mostly for some demanding OPs in /r/RequestABot.

Well, I would say that was a good decision.

> How do you change the level in your app?

I change the log level depending on the environment specified on the configuration file, so for `production` I've set it to `Logger::ERROR`and for `development` `Logger::DEBUG`

So my problem was that I tried to change it with an instance variable of `RedditBot::Bot` instead of module `RedditBot`, which of course won't work.

Thanks.
